### DISCUSSION TITLE
Image Resize Issue

### DISCUSSION THREAD

**Discussion by confettidc** [react-page/react-page#1282] [General]
Hi, would like to see if any advices that we have encountering and kinda critical, we have uploaded images thru react page, while it cannot resize the image, now just can resize by dragging the border of the image cell plugin, but I wish keep the cell plugin untouched but just drag the image to resize (or at least, can provide input of width and height), any suggestion of plug in can share?

thanks a lot
Confetti Se

**Comment by hurricanehunter0702** [comment]
Hi,
I got the same issue before.
I solved it by using react-easy-crop library.
Please try this one.
https://www.npmjs.com/package/react-easy-crop
Cheers.
